src/frontend/js/index.js

```diff
@@ -845,6 +720,14 @@ function meta2options(metadata, selected) {
         .map(([k, v]) => m('option', {value: k, selected: selected === k}, v.name || k))
 }
 
+function scenarios_with(meta, variable) {
+    const v = meta.od_matrices[variable]
+    if (v === undefined)
+        return [];
```

Comment:
```suggestion
        return {}
```

Surely this should always return some sort of object for consistency?